serveur-web/api-v2/app.py

```diff
@@ -32,6 +32,8 @@
 """
 Exemple d'une route simple
 """
+
+
 @app.route("/")
```

Comment:
C'est tu normal les deux lignes vides entre les doc strings et le début de la fonction? Me semble que zero ou une ligne ça serait mieux. Exemple: quand tu fais de la Java doc, habituellement y'a aucune ligne vide entre la doc et la fonction.

Reply:
Le docstring en Python est généralement dans la fonction donc

```py
def method():
    """Mon docstring"""
    pass

# Tu peux alors faire
help(method)
# ou encore
method.__doc__
```

Reply:
Fixed it